### ISSUE TITLE
What happened

### ISSUE THREAD

**Issue by itzmeeugene** [dabusharsystems/Rave-for-GiveWP#1]
Hi, Been looking for this for a long time, please what happened, were you able to develop something?

**Comment by dabusharsystems** [comment]
Yeah... I created a plugin for rave and givewp. Let me look for the source.
